UnityGLTF/Assets/UnityGLTF/Runtime/Scripts/GLTFSceneImporter.cs

```diff
@@ -524,9 +538,10 @@ protected IEnumerator WaitUntilEnum(WaitUntil waitUntil)
 		private async Task LoadJson(string jsonFilePath)
 		{
 #if !WINDOWS_UWP
-			if (IsMultithreaded && _options.ExternalDataLoader.HasSyncLoadMethod)
+			var dataLoader2 = _options.DataLoader as IDataLoader2;
+			if (IsMultithreaded && dataLoader2 != null)
```

Comment:
>dataLoader2 != null [](start = 26, length = 19)

If this targets a new enough version of .NET, you can use this syntax:

_options.DataLoader is IDataLoader2 dataLoader2

Reply:
Not a new enough version of .NET yet (sadly)

---
In reply to: [358533577](https://github.com/KhronosGroup/UnityGLTF/pull/532#discussion_r358533577) [](ancestors = 358533577)